politeiawww/api/v1/v1.go

```diff
@@ -232,6 +232,15 @@ const (
 )
 
 var (
+	// PolicyValidMimeTypes is the list of accepted mime types
```

Comment:
Why is this list repeated in code? There should only be one instance of it.

Reply:
This is the part that had to be temporarily hardcoded. An issue was opened so that we fetch this information of valid mime types from the daemon on startup.